src/app/room/room.cpp

```diff
@@ -18,15 +31,22 @@ std::vector<Treatment> Room::getTreatments() const
 
 std::shared_ptr<Warehouse> Room::getWarehouse() const
 {
-    return ptrToWarehouse_;
+    return warehouse_;
 }
 
 void Room::setRoomAvalaibility(bool avalaible)
 {
     avalaible_ = avalaible;
 }
 
-void Room::setWarehouse(std::shared_ptr<Warehouse>& ptrToWarehouse)
+void Room::setWarehouse(std::shared_ptr<Warehouse>& warehouse)
+{
+    warehouse_ = warehouse;
+}
```

Comment:
Functions that add associations should also make a backward connection to an other class association

Reply:
I wonder if it's needed. As we agreed to have only one instance of warehouse, then it should be initalized in constructor for each room to point to the same instance.
What is your opinion on that? 
 @BaqWin @mBialczak @Aleksiiej @BoskiJanusz